modules/javafx.graphics/src/main/java/javafx/animation/Animation.java

```diff
@@ -976,18 +976,16 @@ public void playFromStart() {
      *  animation.play();<br>
      * </code>
      * <p>
-     * Note: <ul>
-     * <li>{@code play()} is an asynchronous call, the {@code Animation} may not
-     * start immediately. </ul>
-     * <p>
-     * This method must be called on the JavaFX Application thread.
+     * Note: if this method is not called on the JavaFX Application Thread, it is delegated to it automatically.
```

Comment:
We need to leave in the note about it being asynchronous and that that that animation may not start immediately. I recommend to revert that part of the change here and elsewhere.

Reply:
The call is asynchronous (and the animation might not start immediately) only when called from a background thread, so I propose:

```
Note: if this method is not called on the JavaFX Application Thread, it is delegated to it automatically.
In this case, the call is asynchronous and may not happen immediately.
```

Reply:
That seems fine.